components/engine-camel/src/main/java/org/eclipse/dirigible/components/engine/camel/config/CamelEngine.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.dirigible.components.engine.camel.config;
+
+import org.eclipse.dirigible.components.base.artefact.Engine;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CamelEngine implements Engine {
+    @Override
+    public String getName() {
+        return "Process (Camel)";
```

Comment:
We can use the well known term "Integration" here for Camel engine

Reply:
+1